spec/classes/init_spec.rb

```diff
@@ -28,8 +28,8 @@
           :alias  => 'tftp-server',
         })
 
-        if (facts[:operatingsystem] == 'Debian' && facts[:operatingsystemrelease].start_with?('8.')) ||
-           (facts[:operatingsystem] == 'Ubuntu' && facts[:operatingsystemrelease].start_with?('16.'))
+        if (facts[:operatingsystem] == 'Debian' && facts[:operatingsystemrelease].to_i >= 8) ||
```

Comment:
Since 7 is no longer supported that means we can leave out the Debian version check.

Reply:
changed.